packages/react/src/card/docs/overview.mdx

```diff
@@ -16,50 +16,19 @@ Cards can be placed in a 2, 3, or 4-column layout. Cards in a grid layout should
 Consider using other formats such as tables, if more than 12 cards are required.
 
 ```jsx live
-<Flex gap={1} flexDirection="row">
-	<Flex
-		flexDirection="column"
-		flexGrow={1}
-		palette="light"
-		padding={1}
-		background="body"
-	>
-		<Card shadow clickable>
-			<CardInner>
-				<Stack gap={1}>
-					<Heading as="h3" type="h4">
-						<CardLink href="#">Card link</CardLink>
-					</Heading>
-					<Text>
-						Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras non
-						finibus leo.
-					</Text>
-				</Stack>
-			</CardInner>
-		</Card>
-	</Flex>
-	<Flex
-		flexDirection="column"
-		flexGrow={1}
-		palette="dark"
-		padding={1}
-		background="body"
-	>
-		<Card clickable>
-			<CardInner>
-				<Stack gap={1}>
-					<Heading as="h3" type="h4">
-						<CardLink href="#">Card link</CardLink>
-					</Heading>
-					<Text>
-						Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras non
-						finibus leo.
-					</Text>
-				</Stack>
-			</CardInner>
-		</Card>
-	</Flex>
-</Flex>
+<Card shadow clickable>
```

Comment:
I wonder if we should put this in a column still? Especially as the preceding content says "Cards can be placed in a 2, 3, or 4-column layout. Cards in a grid layout should be equal in height and the same type."

Reply:
Works for me!

Reply:
I've updated to simplify it further. More bare-bones now.